### PR TITLE
Adding radio.off and radio.on api

### PR DESCRIPTION
**Pull request by abchatra**
Fixes https://github.com/microsoft/pxt-microbit/issues/4047

While in that code, I changed the default group to zero as well to match micropython as an experimentation. If we hear back issues from the customers for beta, I will revert this back.  This seems a good way to test the hypotheses.

### CODE REVIEW

**Comment by abchatra** [comment]
Micropython: https://microbit-micropython.readthedocs.io/en/latest/radio.html#functions

**Review by riknoll — APPROVED** [review]

**Comment by abchatra** [comment]
I have added simulator support and switched off the radio completely with off api. It needs an on api to switch it back.

**Comment by abchatra** [comment]
@riknoll can you take another peek?

File: libs/radio/radio.cpp [hunk 1]
```diff
@@ -154,6 +206,7 @@ CODAL_RADIO* getRadio() {
 #ifdef CODAL_RADIO        
         if (radioEnable() != DEVICE_OK || NULL == msg) return;
 
+        target_panic(43);
```
[the remark this answers is not in the document]
Comment (abchatra) [reply libs/radio/radio.cpp]: Good catch. I was testing the reviewer's eye. Kidding, forgot to push a commit.

**Pull request merged**